### PR TITLE
Define plain-text version of mails

### PR DESCRIPTION
**Pull request by range-of-motion**

### CODE REVIEW

File: resources/views/emails/reset_password_plain.blade.php [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Use the link below to reset your password.
+
+{{ config('app.url') }}/reset_password?token={{ $token }}
```
Comment (ryangjchandler) [inline resources/views/emails/reset_password_plain.blade.php:3]: Does it not make more sense using the `url()` helper instead of the config value?

Comment (range-of-motion) [reply resources/views/emails/reset_password_plain.blade.php:3]: Didn't know that existed, but I'm a bit confused as to how you'd use it in this case, when there's a query parameter in the URL. 

```php
url('reset_password?token=' . $token)
```

Like this?

Comment (ryangjchandler) [reply resources/views/emails/reset_password_plain.blade.php:3]: Yeah, I think you might also be able to pass an array of query parameters as the second argument, I might be wrong though:

```php
url('/reset_password', ['token' => $token])
```

Comment (range-of-motion) [reply resources/views/emails/reset_password_plain.blade.php:3]: The sample you gave outputs the following–where 123 is the value of `$token`.

`http://localhost:8000/reset_password/123`

Comment (range-of-motion) [reply resources/views/emails/reset_password_plain.blade.php:3]: Would this be an improvement over the existing diff?

```
> echo url('/reset_password?token=' . 123)
http://localhost:8000/reset_password?token=123
```

Comment (ryangjchandler) [reply resources/views/emails/reset_password_plain.blade.php:3]: Probably not - That's annoying that the `url` function doesn't let you define an array of query params.

Comment (range-of-motion) [reply resources/views/emails/reset_password_plain.blade.php:3]: Alright, thanks!

**Pull request merged**